innative/serialize.cpp

```diff
@@ -351,6 +361,9 @@ void Serializer::TokenizeModule(bool emitdebug)
     t.Push(WatToken{ WatTokens::INTEGER, 0, 0, 0, limits.minimum });
     if(limits.flags & WASM_LIMIT_HAS_MAXIMUM)
       t.Push(WatToken{ WatTokens::INTEGER, 0, 0, 0, limits.maximum });
+    if(limits.flags & WASM_LIMIT_SHARED)
+      t.Push(WatToken{ WatTokens::SHARED });
+    // TODO: Should we serialize an UNSHARED token in the other case?
```

Comment:
No, unless other wasm serializers decide to do this (but none of them support threads yet).